app/services/idv/steps/in_person/residential_address_step.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  module Steps
+    module InPerson
+      class ResidentialAddressStep < DocAuthBaseStep
+        STEP_INDICATOR_STEP = :verify_info
+
+        def self.analytics_visited_event
+          :idv_in_person_proofing_residential_address_visited
+        end
+
+        def self.analytics_submitted_event
+          :idv_in_person_proofing_residential_address_submitted
+        end
+
+        def call
+          Idv::InPerson::ResidentialAddressForm::ATTRIBUTES.each do |attr|
+            flow_session[:pii_from_user][attr] = flow_params[attr]
+          end
+
+          if IdentityConfig.store.in_person_capture_secondary_id_enabled
```

Comment:
I wasn't entirely sure if I needed to add the feature flag conditional check here.  After all, the user wouldn't hit this step unless `IdentityConfig.store.in_person_capture_secondary_id_enabled` equaled true.  I decided to err on the side of including it.

Reply:
yeh i think if this step is already behind the feature flag then we probably don't need to do a check again here